### PR TITLE
NOPS-1225 Adding the script to make the DEB package

### PR DESCRIPTION
**Pull request by ashilokhvostov**
Create the build script to make the deb package.

### CODE REVIEW

**Comment by mkostrikin** [comment]
💾

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+UPSTREAM_VERSION=2.4.1
+OWN_VERSION=0.1
```
Comment (mkostrikin) [inline build.sh]: Lets call that env `TRADESHIFT_VERSION`

Comment (ashilokhvostov) [inline build.sh]: Fixed

**Comment by bukowskee** [comment]
👍 
build, run & test
Thanks

**Comment by mkostrikin** [comment]
TODO: xenial support

**Pull request merged**